src/sas/sascalc/dataloader/readers/sesans_reader.py

```diff
@@ -17,158 +17,161 @@
     has_converter = False
 _ZERO = 1e-16
 
+
 class Reader:
     """
     Class to load sesans files (6 columns).
     """
-    ## File type
+    # File type
     type_name = "SESANS"
-    
-    ## Wildcards
+
+    # Wildcards
     type = ["SESANS files (*.ses)|*.ses",
             "SESANS files (*..sesans)|*.sesans"]
-    ## List of allowed extensions
+    # List of allowed extensions
     ext = ['.ses', '.SES', '.sesans', '.SESANS']
-    
-    ## Flag to bypass extension check
+
+    # Flag to bypass extension check
     allow_all = True
-    
+
     def read(self, path):
-        
-#        print "reader triggered"
-        
         """
         Load data file
-        
+
         :param path: file path
-        
+
         :return: SESANSData1D object, or None
-        
+
         :raise RuntimeError: when the file can't be opened
         :raise ValueError: when the length of the data vectors are inconsistent
         """
         if os.path.isfile(path):
             basename = os.path.basename(path)
             _, extension = os.path.splitext(basename)
             if self.allow_all or extension.lower() in self.ext:
```

Comment:
That's certainly easier to follow.  Some of the lines might be turned back into one liners, since they're no longer butting up against the 80 char limit, but I haven't made those changes yet.